osbs/build/build_request.py

```diff
@@ -732,6 +737,11 @@ def render(self, validate=True):
             self.spec.validate()
         super(ProductionBuild, self).render()
 
+        repo_name = git_repo_humanish_part_from_uri(self.spec.git_uri.value)
+        # What about forked repos?
```

Comment:
That's true. I was hoping to keep things human readable if possible, even labels.